CONTRIBUTING.md

```diff
@@ -345,6 +345,18 @@ with clear instructions on how to install the required libraries.
 Furthermore, if your package requires CGO, it MUST be able to compile and operate in a no op mode
 or report a warning back to the collector with a clear error saying CGO is required to work.
 
+### Breaking changes
+
+Whenever possible, we adhere to semver as our minimum standards. Even before v1, we strive to not break compatibility
+without a good reason. Hence, when a change is known to cause a breaking change, it should be clearly marked in the
+changelog, possibly with a line instructing users how to move forward.
+
+We also strive to perform breaking changes in two stages, or using feature flags: 
```

Comment:
Feature flags do not really help with API changes (only with functionality) since they are evaluated at runtime. Alternative can be "build tags"

Reply:
Agreed.  I meant it WRT functionality, or configuration.

Reply:
Maybe we should have an "API breaking changes" section and a "End-user facing breaking changes" section?

Reply:
I added a paragraph about user-facing changes and feature flags, removing it from this paragraph here. Let me know if you think this should be on its own section.

Reply:
Looks good to me like this!